src/data/community-events.json

```
@@ -332,6 +332,15 @@
     "startDate": "2022-09-01",
     "endDate": "2022-09-30"
   },
+  {
+    "title": "Devcon",
+    "to": "https://devcon.org/en/#road-to-devcon",
```

Devcon site is multi-lingual, removing the absolute /en/ path so that users are directed to their own language, if it is available.

```suggestion
    "to": "https://devcon.org/",
```